core/src/edu/cornell/gdiac/nightbite/Assets.java

```diff
@@ -188,6 +191,9 @@
      * LOADED ASSETS
      */
     public static FilmStrip[] PLAYER_FILMSTRIPS;
+    public static TextureRegion[] WOK_LIST;
```

Comment:
Assume that we only have one human-controlled player, and that is the only in-game sprite that has a wok, shadow, and arrow